schema/models_directory/pb/at/disp/dft_d.json

```diff
@@ -0,0 +1,52 @@
+{
+    "schemaId": "models-directory-at-disp-dft-d",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Grimme-type empirical dispersion model for DFT (DFT-D)",
+    "allOf": [
+        {
+            "$ref": "../disp.json"
+        }
+    ],
+    "properties": {
+        "type": {
+            "enum": [
+                {
+                    "name": "Empirical dispersion DFT-D",
+                    "slug": "empirical-dft-d"
+                }
+            ]
+        },
+        "subtype": {
+            "enum": [
+                {
+                    "name": "DFT-D2",
+                    "slug": "dft-d2"
```

Comment:
Maybe we should add a link to the description above explaining where we took the naming from